### PR TITLE
Prevent duplicated event dispatching

### PR DESCRIPTION
**Pull request by hata6502**
https://github.com/codex-team/editor.js/issues/1174

### CODE REVIEW

File: src/components/core.ts [hunk 1]
```diff
@@ -185,7 +185,6 @@ export default class Core {
       a: true,
     } as SanitizerConfig;
 
-
```
Comment (hata6502) [inline src/components/core.ts:188]: `yarn lint:fix`

File: src/components/modules/blockSelection.ts [hunk 2]
```diff
@@ -175,7 +175,7 @@ export default class BlockSelection extends Module {
          *
          * Prevent such actions if focus is not inside the Editor
          */
-        if (!BlockManager.currentBlock) {
+        if (!event.target.closest(`.${this.Editor.UI.CSS.editorWrapper}`)?.isEqualNode(this.Editor.UI.nodes.wrapper)) {
```
Comment (hata6502) [inline src/components/modules/blockSelection.ts:178]: When multiple Editor.js instances are launched, `BlockManager.currentBlock` can't be changed to null by focusing other instances.

Comment (khaydarov) [reply src/components/modules/blockSelection.ts:178]: maybe it makes sense to create a function inside UI module that gets event and checks if event's target is under the current editors instance wrapper?

Comment (hata6502) [reply src/components/modules/blockSelection.ts:178]: Exactly, this method seems DOM hacking!
Sorry, this problem has been resolved by https://github.com/codex-team/editor.js/pull/1431

File: src/components/modules/paste.ts [hunk 3]
```diff
@@ -439,6 +439,13 @@ export default class Paste extends Module {
   private handlePasteEvent = async (event: ClipboardEvent): Promise<void> => {
     const { BlockManager, Toolbar } = this.Editor;
 
+    if (
+      !(event.target instanceof Element) ||
+      !event.target.closest(`.${this.Editor.UI.CSS.editorWrapper}`)?.isEqualNode(this.Editor.UI.nodes.wrapper)
+    ) {
+      return;
+    }
+
```
Comment (hata6502) [inline src/components/modules/paste.ts:448]: And... This fix is also resolved by https://github.com/codex-team/editor.js/pull/1431
This PR is invalid. 
Sorry, I'll close it!

**Comment by hata6502** [comment]
These fixes are resolved by #1431
Sorry, I'll close this PR because it is invalid.

**Pull request closed without merging**